examples/flutter_example/lib/main.dart

```diff
@@ -55,12 +55,19 @@ class _MyHomePageState extends State<MyHomePage> {
   @override
   void initState() {
     super.initState();
-    One.initializeOne(SITE_KEY, TOUCHPOINT, API_KEY, SHARED_SECRET, USER_ID, HOST, false);
-    One.setThunderheadLogLevel(true);
+    // One.optOut(true);
 
     // Calling this in `initState()` for demonstration purposes.
-    // When to send these Interaction requests should be aligned with your business use cases rather than follow exactly this code placement.
-    One.sendInteraction("/home", null);
+    One.initializeOne(SITE_KEY, TOUCHPOINT, API_KEY, SHARED_SECRET, USER_ID, HOST, false).then((_) {
+      One.setThunderheadLogLevel(true);
+        
+      // When to send these Interaction requests should be aligned with your business use cases rather than follow exactly this code placement.
+      One.sendInteraction("/home").then((response) {
+        print('Interaction response tid = ${response[oneResponseTidKey]}');
```

Comment:
Can you point me to where these constants are defined? I might not be seeing them in this big diff.

Reply:
It's in the one_sdk_flutter.dart file.  https://github.com/thunderheadone/one-sdk-flutter/blob/814a74ca9b8f7353adeec457c50f865880de9021/lib/one_sdk_flutter.dart#L5